src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepository.java

```diff
@@ -88,11 +88,6 @@
             "    WHERE OHP.OFFENDER_BOOK_ID = O.OFFENDER_BOOK_ID" +
             ") " +
             "AND NOT EXISTS (" +
-            "    SELECT 1 FROM OFFENDER_ALERTS OA" +
-            "    WHERE OA.OFFENDER_BOOK_ID = O.OFFENDER_BOOK_ID" +
-            "    AND OA.ALERT_CODE = 'XTACT'" +
-            ") " +
```

Comment:
Dropping this filter on XTACT since the aim is to do a check on ALL of the offender's alerts in the Data Compliance Service